### ISSUE TITLE
Syntax highlighting of .moor files in VSCode (almost perfect)

### ISSUE THREAD

**Issue by JCKodel** [simolus3/moor#473]
To get syntax highlighting (at least the colors), I've configurated my VS code as this:

```json
{
	"[sql]": {
		"editor.formatOnSave": false,
		"editor.formatOnPaste": false,
		"editor.formatOnType": false
	},
	"files.associations": {
		"**/*.moor": "sql",
		"*.moor": "sql"
	}
}
```

This will result in:
![image](https://user-images.githubusercontent.com/379339/78473591-3b895d80-7718-11ea-95d6-5da54c75380b.png)

It works, but it shows errors because it tries to parse it as an MSSQL script.

When I set the language mode to sql.none, it works as intended:

1) Click here
![image](https://user-images.githubusercontent.com/379339/78473613-65428480-7718-11ea-9730-de438f039221.png)

2) Change to none here:
![image](https://user-images.githubusercontent.com/379339/78473625-79868180-7718-11ea-8486-26edd6c8b18b.png)

The, I get the .moor file with coloring and autocomplete (throught the moor extension), without any errors (as intended).

![image](https://user-images.githubusercontent.com/379339/78473638-9f138b00-7718-11ea-9c90-8fbc5129905f.png)

My question is: how to automatically configure the .moor extension to sql language mode, with NONE pre selected?

**Comment by North101** [comment]
I believe this is what you're looking for: https://scottaddie.com/2016/04/10/file-to-language-associations-in-visual-studio-code/

**Comment by simolus3** [comment]
After https://github.com/Dart-Code/Dart-Code/pull/2203 gets released (might take some time, the underlying apis are still experimental in VS Code) moor will be able to contribute its own highlighting.

You can already use moor's [IDE](https://moor.simonbinder.eu/docs/using-sql/sql_ide/) with the Dart extension for VS Code. This will enable error reporting, folding, outline and (partly) navigation and auto-complete. It's still a bit buggy, but it might help.

**Comment by JCKodel** [comment]
> I believe this is what you're looking for: https://scottaddie.com/2016/04/10/file-to-language-associations-in-visual-studio-code/

Not really. I already did all, except disable the MSSQL parsing automatically.

> You can already use moor's [IDE](https://moor.simonbinder.eu/docs/using-sql/sql_ide/) with the Dart extension for VS Code. This will enable error reporting, folding, outline and (partly) navigation and auto-complete. It's still a bit buggy, but it might help.

I'm already doing this (and I really don't care much for intellisense). But moor's IDE don't have syntax highlighting (it treats the file as text, with all text white).

**Comment by JCKodel** [comment]
Solved removing all sql extensions from VSCode (it will still recognize sql files with color syntax highlighting, but won't try to parse it).

As I use Microsoft SQL Server Management Studio or Azure Data Studio for SQL stuff, that plugin will not be missed.

**Comment by listepo** [comment]
Any news about vscode extension for .drift files?
